tests/main/lxd-postrm-purge/prep-snapd-in-lxd.sh

```diff
@@ -2,6 +2,11 @@
 
 set -ex
 
+# TODO: Remove the refresh once the issue https://github.com/lxc/lxd/issues/10079 is release to 4.0/candidate
+# Make sure the lxd snap is updated before removing it
+snap refresh lxd --channel=latest
```

Comment:
is this mean to go to `latest/stable` or just `latest/<whatever-the-current-risk-is>` ?